study.md

```diff
@@ -17,7 +17,9 @@ material. Instead, digest what you've read and repeat it in your own voice.
 Describe three examples of differences between Ruby and JavaScript.
 
 ```md
-<!-- your answer here -->
+Ruby has no semicolons
```

Comment:
It doesn't have none, but it doesn't use them at the end of every line like javascript does.